launcher.sh

```diff
@@ -73,6 +73,6 @@ cd $datadir
 # snapshot.
 latest=$(aws s3 ls "s3://${snapshots_bucket}/${chain}/${client}/${dockerhub_tag}/" | awk '{print $4}' | tail -1)
 aws s3 cp "s3://${snapshots_bucket}/${chain}/${client}/${dockerhub_tag}/${latest}" - | pv | /zstd/zstd --long=31 -d | tar -xf -
-docker run -d --name $client -v /$client/:/$client $docker_port_mappings ${dockerhub_repo}:${dockerhub_tag} $docker_cmd --datadir $datadir
+docker run -d --name $client -v ${datadir}/${datadir} -v /$client/:/$client $docker_port_mappings ${dockerhub_repo}:${dockerhub_tag} $docker_cmd --datadir $datadir
```

Comment:
Yikes. That's quite a bug. Thanks for catching it!

I think maybe it needs to be ```-v ${datadir}/:/$client```, though? Otherwise I think the docker command (which makes assumptions about the mounted data dir) won't do the right thing...

Reply:
Since erigon look into whatever we pass into `datadir`, which is `$datadir` ,so we need to have `$datadir` exist inside the docker container?

Reply:
oh yeah. I forgot we made that change! In that case, I think ditching the `-v /$client/:/$client` part is what is required here

Reply:
I'm not convinced this is going to work with erigon, though... Might need a bit more thought